### DISCUSSION TITLE
Timeline or rough estimate for Ubuntu 22.04 virtual environments?

### DISCUSSION THREAD

**Discussion by lanctot** [actions/virtual-environments#5427] [General questions] [answered]
(Apologies if this is already contained somewhere that I missed. If so, please point me to the docs.)

I'm mostly new-ish to Github Actions and we have been using it for CI in our project. It has worked tremendously well, and I am super grateful for all the effort and compute donated to maintaining healthy infrastructure for open source software development. So, thank you one thousand times over!

I'm just curious roughly when the virtual environments become available after the official releases. This is mostly to help with planning of our project's timelines. I'd be happy with just a rough estimate (maybe, based on the past two versions?) and won't hold anybody to their word :) Thanks!

**Comment by miketimofeev** [comment, marked as answer]
@lanctot Hi! We are working on adding Ubuntu 22 image in a preview state (which means not all the software will be available from the beginning). No exact ETA, for now, but most likely the image will be ready early in May.

**Reply by lanctot** [reply]
Great, thanks! Appreciate the quick response.

**Reply by MSP-Greg** [reply]
gcc 12.1 has been released.  Maybe consider installing it, but not making it the default gcc?

**Reply by miketimofeev** [reply]
@MSP-Greg feel free to submit a PR
